files/gen_chkrootkit_expected_output.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+LOG_DIR=/var/log/chkrootkit
+
+eval /usr/sbin/chkrootkit -q > $LOG_DIR/log.today.raw 2>&1
```

Comment:
In this particular case, I don't think it does.
The cron script uses, but I'm not sure why.

I can remove the eval.